<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Show</class>
 <widget class="QMainWindow" name="Show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(67, 67, 67);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(64, 64, 64);
background-color: rgb(187, 134, 252);</string>
    </property>
    <property name="text">
     <string>   enter animal name and animal nike name to show it.</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>661</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Arial&quot;;
background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>340</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(64, 64, 64);
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string>animal name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Arial&quot;;
background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(64, 64, 64);
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string>animal nike name :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_NikeName">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Arial&quot;;
background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>304</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButtonShow">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(3, 218, 197);
font: 75 14pt &quot;Arial&quot;;
color: rgb(15, 15, 15);</string>
         </property>
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_back">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(3, 218, 197);
font: 75 14pt &quot;Arial&quot;;
color: rgb(15, 15, 15);</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(3, 218, 197);
font: 75 14pt &quot;Arial&quot;;
color: rgb(15, 15, 15);</string>
       </property>
       <property name="text">
        <string>Show all animal</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
